src/extensions/generateInstanceId.ts

```diff
@@ -12,6 +12,9 @@ export async function generateInstanceId(projectId: string, modName: string): Pr
     if (_.get(instanceRes, "error.code") === 404) {
       return modName;
     }
+    if (_.get(instanceRes, "error.code") === 403) {
+      throw new FirebaseError(_.get(instanceRes, "error.message"));
```

Comment:
What do you think about putting `res.error.message` (like this) into the `FirebaseError` message below? That way, if there's any message, we always show it?